### ISSUE TITLE
New tesseract-ocr versions

### ISSUE THREAD

**Issue by MagicGreenDragon** [SubtitleEdit/subtitleedit#2966]
Hi.
I see that there is a new version of tesseract-ocr (stable and beta).

https://github.com/tesseract-ocr/tesseract/releases/tag/4.0.0-beta.3
https://github.com/tesseract-ocr/tesseract/releases/tag/3.05.02

Seeing the changelogs they just seems only bugfix releases, so the update should be safe.

Can you update them?

Thanks in advance!

**Comment by niksedk** [comment]
Latest SE beta already includes tesseract 4: https://github.com/SubtitleEdit/subtitleedit/releases/download/3.5.6/SubtitleEditBeta.zip

**Comment by MagicGreenDragon** [comment]
The latest beta includes v4.0.0-beta.1, but the latest (released 5 days ago) is v4.0.0-beta.3

```
tesseract 4.0.0-beta.1
 leptonica-1.74.4 (Mar 28 2018, 17:19:47) [MSC v.1913 LIB Release x86]
  libgif 5.1.4 : libjpeg 6b (libjpeg-turbo 1.5.3) : libpng 1.6.34 : libtiff 4.0.9 : zlib 1.2.11

 Found AVX
 Found SSE
```

**Comment by niksedk** [comment]
Beta updated via 291bbd7c4421bdfa6b159e2926cced00462b77bc : https://github.com/SubtitleEdit/subtitleedit/releases/download/3.5.6/SubtitleEditBeta.zip
